.travis/linux/install.sh

```diff
@@ -5,6 +5,7 @@ set -ex
 go get golang.org/x/tools/cmd/cover
 go get github.com/mattn/goveralls
 go get github.com/gonum/floats
+go get github.com/cznic/cc
```

Comment:
`github.com/gonum/internal/binding` is not installed from here since we get it for free with `blas` and `floats` via `asm`. Does anyone want me to be explicit here?